packages/block-library/src/query-pagination/query-pagination-label-control.js

```diff
@@ -9,9 +9,7 @@ export function QueryPaginationLabelControl( { value, onChange } ) {
 		<ToggleControl
 			__nextHasNoMarginBottom
 			label={ __( 'Show label text' ) }
-			help={ __(
-				'Toggle off to hide the label text, e.g. "Next Page".'
-			) }
+			help={ __( 'Make label text visible to the users.' ) }
```

Comment:
Now the help text doesn't really provide much help over the label and could even be removed.

Reply:
Should I change it to - `Make label text visible, e.g. "Next Page"?

Reply:
That would provide more context 👍